src/frontend/handlers/accessibility.ts

```diff
@@ -0,0 +1,29 @@
+import { PageParamsGetter, PageTitleGetter} from "../pages";
+
+
+export type AccessibilityParams ={
+  organizationName: string;
+  statementScope: string;
+  wcagVersion: string;
+  websiteOrApp: string;
+  statementDate: string;
+  statementPreparationMethod: string;
+  statementLastReviewDate: string;
+}
+
+export const accessibilityParamsGetter: PageParamsGetter<{},AccessibilityParams> = async () => {
+  // Fetch data from your data source, database, or any other source
+  // For porpuse, we'll just return static data
+  return {
+    organizationName: 'Government Digital Service (GDS)',
+    statementScope: 'GOV.UK BTM is run by the Government Digital Service (GDS), part of the Cabinet Office. We want as many people as possible to be able to use our BTM services.',
```

Comment:
"We want as many people as possible to be able to use this website. For example, that means you should be able to:

change colours, contrast levels and fonts using browser or device settings
zoom in up to 400% without the text spilling off the screen
navigate most of the website using a keyboard or speech recognition software
listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
We’ve also made the website text as simple as possible to understand.

[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability."